src/components/interfaces/HMI_API.xml

```diff
@@ -4350,6 +4350,13 @@
     <param name="state" type="Common.DriverDistractionState" mandatory="true">
       <description>See DriverDistractionState. </description>
     </param>
+     <param name="lockScreenDismissalEnabled" type="Boolean" mandatory="false">
+      <description>
```

Comment:
@IGapchuk Check indents

Reply:
@AByzhynar Fixed in 22121736e2e4ec93cfa7a55a0e295df8f2a40223.